src/app/menu/menu.component.scss

```diff
@@ -103,7 +103,7 @@
 }
 
 .prevent-inner-circle {
-  background: $prevent-color;
+    background: $prevent-color;
```

Comment:
This is not supposed to be like this, the colors are supposed to be generic.  
This kinds of defeats the purpose, same case with the whole file.

Reply:
My old comment said the same and its lost due to rebase..